### PR TITLE
Fix help for device run and emulate android commands

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

**Comment by teobugslayer** [comment]
:+1:

**Comment by Fatme** [comment]
@ikoevska

File: resources/help.txt [hunk 1]
```diff
@@ -329,7 +331,11 @@ Usage:
 
 Run the selected application on device.
 <ApplicationId> is the application id as listed from $tns device list-applications
```
Comment (ikoevska) [inline resources/help.txt]: $ tns ???

File: resources/help.txt [hunk 2]
```diff
@@ -329,7 +331,11 @@ Usage:
...
-
+Prerequisites:
+Before running the application on device, verify that you have met the following requirements.
+    You are running the Telerik AppBuilder CLI on OS X.
+    You have installed Xcode.
+    Your application is built in debug.
```
Comment (ikoevska) [inline resources/help.txt]: Your application is built with the debug build configuration.

File: resources/help.txt [hunk 3]
```diff
@@ -673,7 +679,7 @@ Options:
             To list the available Android virtual devices, run $ android list avd
     --geny - Sets the Genymotion virtual device on which you want to run your app. 
             You can set only one device at a time.
-            To list the available Android virtual devices, run $ android list avd        
+            To list the available Android virtual devices, run $ Genyshell's `devices list` command.
```
Comment (ikoevska) [inline resources/help.txt]: $ genyshell -c "devices list"

**Comment by Fatme** [comment]
run ci

**Comment by tytodorov** [comment]
:-1: This help is old. See http://teampulse.telerik.com/view#item/283764 new update

**Pull request merged**